### PR TITLE
Mihai ex1

### PR DESCRIPTION
**Pull request by mihaibota**
finished ex 1
- added some files in the structure

### CODE REVIEW

File: features/store_tests/auth.feature [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+#   ====== Checks various login errors ====== Ex 1
+@authentication
+Scenario: Check if wrong credentials login is signaled properly (various cases)
+Then I click the 'sign in' button
+Then on the Auth page I enter the wrong credentials and look for the errors
+| mail           | pass      | errors                     |
+| mihai.bota     | match     | Invalid email address.     |
+|                | match     | An email address required. |
+| mihaita@gg.com | invalid   | Authentication failed.     |
+| mihaita@gg.com |           | Password is required.      |
```
Comment (lucksnail) [inline features/store_tests/auth.feature]: Use the rubymine code 'auto-arange' functionality as often as possible. Use it in all files you create, before pushing your last commit.
Shortcut keys:  -  Ctrl + Alt + L

File: features/store_tests/auth.feature [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+#   ====== Checks various login errors ====== Ex 1
+ @authentication
+  Scenario: Check if wrong credentials login is signaled properly (various cases)
+    Then I click the 'sign in' button
+    When I enter the mihai.bota mail adress in the email field
+    And I enter the pass password in the password field
+    And I click the "Sign in" button on the Authentification page
+    Then The Invalid email address. error should be displayed
+
```
[the remark this answers is not in the document]
Comment (mihaibota) [reply features/store_tests/auth.feature]: Thanks for the input! Removed the unnecessarily scenario.

File: features/step_definitions/auth_page_steps.rb [hunk 3]
```diff
@@ -2,4 +2,35 @@
...
+  @authentication_page.sing_in_btn
+end
+
+Then(/^The (.*) error should be displayed$/) do |error|
+  if @authentication_page.error_message_specific_txt != error
+    puts "<" + @authentication_page.error_message_specific_txt + "> was discovered instead of: " + error
+  end
+  @authentication_page.error_message_specific_txt.should == error
+end
+
```
[the remark this answers is not in the document]
Comment (mihaibota) [reply features/step_definitions/auth_page_steps.rb]: deleted unused steps from auth_page_steps.rb

File: features/store_tests/auth.feature [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+#   ====== Checks various login errors ====== Ex 1
```
[the remark this answers is not in the document]
Comment (mihaibota) [reply features/store_tests/auth.feature]: removed comments from auth.feature file.

File: features/store_tests/auth.feature (hunk 1 above)
Comment (mihaibota) [reply features/store_tests/auth.feature]: used the shortcut: Ctrl + Alt + L for re-formatting the code in the changed classes.

**Pull request merged**